engine/src/witness/arb.rs

```diff
@@ -173,3 +173,97 @@ where
 
 	Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+
+	use std::path::PathBuf;
+
+	use cf_primitives::AccountRole;
+
+	use crate::{
+		settings::{NodeContainer, WsHttpEndpoints},
+		state_chain_observer,
+		witness::common::epoch_source::EpochSource,
+	};
+
+	use futures::FutureExt;
+	use utilities::{
+		logging::LoggingSettings, task_scope::task_scope,
+		testing::new_temp_directory_with_nonexistent_file,
+	};
+
+	use super::*;
+
+	#[ignore = "requires a running localnet"]
+	#[tokio::test]
+	async fn run_arb_witnessing() {
+		let _start_logger_server_fn = Some(
+			utilities::logging::init_json_logger(LoggingSettings {
+				span_lifecycle: false,
+				command_server_port: 6666,
+			})
+			.await,
+		);
+
+		task_scope(|scope| {
+			async move {
+				let (state_chain_stream, _unfinalised_state_chain_stream, state_chain_client) =
+					state_chain_observer::client::StateChainClient::connect_with_account(
+						scope,
+						"ws://localhost:9944",
+						PathBuf::from("/Users/kylezs/Documents/cf-repos/chainflip-backend/localnet/init/keys/bashful/signing_key_file").as_path(),
```

Comment:
probably a local path that sneaked in there. There is some others.

Reply:
it's an ignored test that will need local paths changed anyway so i left it